taxonomy/algolia/utils.py

```diff
@@ -49,13 +129,35 @@ def fetch_jobs_data():
     Returns:
         (list<dict>): A list of dicts containing job data.
     """
+    qs = Job.objects.exclude(name__isnull=True)
```

Comment:
question: are there any jobs in our system that do not have a name?

Reply:
Not sure about it. This line comes from the previous commit. I have placed it at the top of function to avoid re-query to the database as the same queryset is needed to fetch skills which are needed in job recommendation calculations.